rs-libc/src/asm/x86_64/sqrt.S

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Is this file necessary? That's surprising to me. What happens if you call `f64::sqrt` in Rust?

Reply:
I'll have to try, but it was up there with the other missing math functions.